src/app/child-dev-project/children/children.service.ts

```diff
@@ -4,7 +4,7 @@ import { Child } from './model/child';
 import { EntityMapperService } from '../../core/entity/entity-mapper.service';
 import { AttendanceMonth } from '../attendance/model/attendance-month';
 import { Database } from '../../core/database/database';
-import { Note } from '../notes/model/note';
+import { NoteModel } from '../notes/note.model';
```

Comment:
Any particular reason to rename the `Note` class to `NoteModel`? I think we should keep this consistent for all Entity classes and personally favour the short `Note` but we can put this up for discussion

Reply:
I have done this to distinguish the new `NoteModel` from the old `Note` during refactoring from a component to a module. I kept the change to make it clear that this was a model. Since I agree that this is no longer needed, I will revert the changes